### DISCUSSION TITLE
Activities template for projects

### DISCUSSION THREAD

**Discussion by racortesl** [kimai/kimai#2307] [General]
Hello there,

I've been using Kimai (1.7) for a while at work and we ran into a "problem". We established different types of project, every one with the same default activities, but every time we create a new project in Kimai, we have to create all the activities one by one all over again. This could be done more efficiently if there were an option to generate templates for the project's activities.

For example, Project1 is a typeA project, which have the activities A1, A2 and A3. Project2 is a typeB project, with activities B1, B2, B3 and B4, and Project3 is a typeA project too with the same activities than Project1.

It wold be great if this could be implementes in a further version of Kimai.

**Comment by kevinpapst** [comment]
1.7 is one year old! You can copy Projects, update to a recent version.

**Reply by racortesl** [reply]
oooh I looked for something like that in the last demo, but i didn't try to copy a project. Nice.

Thanks!
